src/xr/xr-manager.js

```diff
@@ -683,6 +685,9 @@ class XrManager extends EventHandler {
         }
 
         const pose = frame.getViewerPose(this._referenceSpace);
+
+        if (!pose) return false;
+
         const lengthNew = pose ? pose.views.length : 0;
```

Comment:
this line can be simplified now that pose is always valid. It could be
`const lengthNew = pose.views.length;`

Reply:
I've committed the suggested change.